pom.xml

```diff
@@ -23,10 +23,11 @@
         <dependency>
             <groupId>junit</groupId>
             <artifactId>junit</artifactId>
-            <version>4.12</version>
+            <version>4.13.2</version>
```

Comment:
You don't have to change junit version

https://mate-academy.github.io/jv-program-common-mistakes/java-core/solid/fruit-shop-tests#dont-use-any-other-version-of-junit